.yamllint.yaml

```diff
@@ -1,9 +1,40 @@
 extends: default
 
 rules:
+  braces:
+    min-spaces-inside: 0
+    max-spaces-inside: 1
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 0
+
+  brackets:
+    min-spaces-inside: 0
+    max-spaces-inside: 1
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 0
+
+  comments-indentation:
+    level: error
+
+  document-end:
+    present: false
+
   document-start:
+    level: error
     present: false
+
+  empty-lines:
+    max: 1
+
+  indentation:
+    spaces: 2
+
   line-length: disable
+
+  quoted-strings:
+    quote-type: single
```

Comment:
If we were to set `double` here, wouldn't the diff be smaller?

Reply:
I don't know, there are also a lot of single quotes. (btw, there is no option to auto fix, so I do it manually)

Reply:
Just tried locally:

- With double quotes enforced, the diff is `27 files changed, 210 insertions(+), 179 deletions(-)`.
- With single quote enforces, the diff is `27 files changed, 186 insertions(+), 155 deletions(-)`.

Single quotes seems to produce the smaller diff let's keep it.